src/app/components/Covid19RedirectWarning.tsx

```diff
@@ -0,0 +1,28 @@
+import { Message } from 'franklin-sites';
+import { useState } from 'react';
+
+import style from './styles/warning-message.module.scss';
+
+const reFromCovid19Portal = /fromCovid19Portal=true/;
+
+const Covid19DeploymentWarning = () => {
+  const [dismissed, setDismissed] = useState(false);
+  return (
+    <>
+      {window.location.search.match(reFromCovid19Portal) &&
+      !LIVE_RELOAD &&
+      !dismissed ? (
+        <Message
+          className={style['warning-message']}
+          level="warning"
+          onDismiss={() => setDismissed(true)}
+        >
+          You have been redirected from the COVID-19 Portal which has been shut
+          down.
```

Comment:
Not sure if there's a smoother verb than "shut down" in English.
I would also add something like "All the entries in the portal are now fully integrated into UniProtKB".

Reply:
Discontinued?

Reply:
You have been redirected from the COVID-19 Portal which is now obsolete as all entries from the portal have been fully integrated into UniProtKB.

Reply:
Yep, that suggestion sounds good